### ISSUE TITLE
Project: TagMyFav is a bookmarking manager integrated within Microsoft 365 that incorporates an advanced AI to enhance collaboration and research within teams.

### ISSUE THREAD

**Issue by AvishkaramTech** [microsoft/hack-together-teams#75]
### Project name

TagMyFav-AI

### Description

TagMyFav, the innovative bookmarking manager in Microsoft 365 has an advanced AI that transforms the way teams collaborate and conduct research. 

Seamlessly integrated within Microsoft Teams, TagMyFav now enables users to engage in conversation with the AI and request web links, which are swiftly curated and enlisted during the conversation. Users are able to bookmark those link with "Add Bookmark" button available with every individual link TagMyFav AI provided. 

With this powerful AI capability, TagMyFav revolutionizes knowledge management, research, and development for teams across diverse industries, unleashing boundless possibilities. 

**How to ask the TagMyFav AI for links:** 

To ask the **TagMyFav AI** for links, simply type a query. 

For example, you could type "provide me with links of best marketing research papers" or "what are the links of top open source software projects." The AI will then search for and return links that match your query. 

Here are some examples of queries that you can use to ask the **TagMyFav AI** for links:

- Best marketing research papers
- Top open source software projects
- Latest news articles about AI
- Most popular books about machine learning
- Upcoming conferences on data science

The **TagMyFav AI** can understand a wide range of queries, so you can be creative with your requests. The more specific your query, the more likely the **TagMyFav AI** is to return relevant results.

### Project Repository URL

https://github.com/AvishkaramTech/TagMyFav-AI

### Project video

https://www.youtube.com/watch?v=l-B_sRG0IVg

### Team members

pallav@avishkaram.com, nischit.gautam@avishkaram.com

**Comment by svrooij** [comment]
Can we see how you did this? I'm really curious to see what others build. I see you also made more awesome videos on your [youtube channel](https://www.youtube.com/@tagmyfav6308), they look so professional, I need to hire your video editor 😉

**Comment by waldekmastykarz** [comment]
Thank you for your submission @AvishkaramTech! Could it be that your repository is still private? If so, could you please make it public so that we'll be able to review your submission?

**Comment by AvishkaramTech** [comment]
@waldekmastykarz We can't make our repo public, TagMyFav-AI is an extension to existing app  with the AI feature and source code is private. We have extended the feature to the [TagMyFav](https://appsource.microsoft.com/en-US/product/web-apps/avishkaramtechnologiespvtltd1610699900335.tagmyfav?exp=kyyw&tab=Overview) app.

**Comment by aycabas** [comment]
Hi @AvishkaramTech, I see your other submission in the list, you already provided public repo and video there: https://github.com/microsoft/hack-together-teams/issues/84

If this is a duplicate submission, please let us know, so we can close this one. Thanks for submitting your amazing project 🙂

**Comment by AvishkaramTech** [comment]
@aycabas Yeah we had submitted the new submission but that is another project ([AI Summary Bot [Sambhasha - AI]](https://github.com/microsoft/hack-together-teams/issues/84)). Regarding this project (TagMyFav-AI), this is the extension to the exiting app and we can't make it public. So can we add one of you as a collaborator to this project so you can review and mark as submission.

**Comment by aycabas** [comment]
I'm afraid we have several people in our judge board who need to review the project, not only us. So it'd be challenging for our judge board to review this in private. I'll tag this project as invalid for now but if there is any way you can share this publicly, let us know. Thanks for all your work in the hackathon @AvishkaramTech, looking forward to checking out the other project you submitted 🥳

**Comment by AvishkaramTech** [comment]
@aycabas Yeah we have separated the code and now made it as public. You can review it rather than marking it as invalid.

**Comment by waldekmastykarz** [comment]
Hey @AvishkaramTech the repo link still returns 404. Have you made it public?

Also, I noticed sulav@avishkaram.com listed on both this project and #84. Following our rules, each participant is allowed to submit only one entry. Could you please ask him to choose which project he'd like to be a member on? 🙂

**Comment by AvishkaramTech** [comment]
Hi @aycabas, sorry. In the mean time, we were pushing the code and you might encounter with the 404 error. Its available now. Can you please review it? In addition, I have edited the team members in both of the submitted project. Please verify it as well.
Thank You.

**Comment by AvishkaramTech** [comment]
@waldekmastykarz Thank you for you co-operation. :)

**Comment by waldekmastykarz** [comment]
Pleasure is all ours! If I could ask one more favor, would you be willing to fill in a feedback survey? It would help us improve next time and we'd appreciate it very much: https://aka.ms/hack-together-teams/survey.

**Comment by aycabas** [comment]
Thanks @AvishkaramTech, excited to see what you've built! 🙂

**Comment by aycabas** [comment]
Hi @AvishkaramTech, can you please make sure that [pallav@avishkaram.com](mailto:pallav@avishkaram.com), [nischit.gautam@avishkaram.com](mailto:nischit.gautam@avishkaram.com) are both registered to the hackathon. I cannot find their details in the registration and if they don't register, they won't be eligible for the digital badges. They can complete the registration from here: https://aka.ms/hack-together-teams/register
